fitbenchmarking/results_output.py

```diff
@@ -5,14 +5,17 @@
 from __future__ import (absolute_import, division, print_function)
 from collections import OrderedDict
 import copy
+from jinja2 import Environment, FileSystemLoader
 import logging
 import os
 import pandas as pd
 import pypandoc
+import webbrowser
 
 from fitbenchmarking.resproc import visual_pages
 from fitbenchmarking.utils import create_dirs
 from fitbenchmarking.utils.logging_setup import logger
```

Comment:
flake8 again complains about logger not being used

Reply:
removed logger